index.html

```diff
@@ -678,8 +678,59 @@ <h4>Search</h4>
+		    </span>
+        <p class="ednote" title="Normative JSON Block">Should we have the previous sentence splitted into three different assertions?.</p>
+    		
+        <section id="jsonpath-semantic" class="normative">
+          <h4>Syntactic search: JSONPath</h4>
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-method"> The API MUST allow searching TDs using an HTTP <code>GET</code> request.</span> 
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-parameter"> The request MUST contain a valid JSON Path [[JSON Path]] as searching parameter.</span>
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-response">  A successful response MUST have 200 (OK) status, contain <code>application/json</code> Content-Type header, and in the body a set of complete TDs or a set of TD elements.</span>
+          The syntactic search with JSON Path is specified as `searchJSONPath` property in <a href="#directory-thing-description">Directory's Thing Description</a>.
+
+	  <p>List of errors:
+            <ul>
+              <li>400(Bad Request): if the provided <code>JSON Path</code> contains syntax errors.</li>
+              <li>401 (Unauthorized): No authentication.</li>
+              <li>403 (Forbidden): Insufficient rights to the resource.</li>
+            </ul>
+          </p>
+        </section>
+	<section id="xpath-semantic" class="normative">
+    	<h4>Syntactic search: XPath</h4>
+    	<span class="rfc2119-assertion" id="tdd-search-xpath-method">The API MUST allow searching TDs using an HTTP <code>GET</code> request.</span>
+    	<span class="rfc2119-assertion" id="tdd-search-xpath-parameter"> The request MUST contain a valid XPath [[xpath-31]] as searching parameter.</span>
+    	<span class="rfc2119-assertion" id="tdd-search-xpath-response"> A successful response MUST have 200 (OK) status, contain <code>application/json</code> Content-Type header, and in the body a set of complete TDs or a set of TD elements.</span>
+	The syntactic search with XPath is specified as `searchXPath` property in <a href="#directory-thing-description">Directory's Thing Description</a>.
+                            
+	 <p>List of errors:
+    	  <ul>
+    	   <li>400(Bad Request): if the provided <code>XPath</code> contains syntax errors.</li>
+           <li>401 (Unauthorized): No authentication.</li>
+           <li>403 (Forbidden): Insufficient rights to the resource.</li>
+    	  </ul>
+    	 </p>
+        </section>
+	<section id="search-semantic" class="normative">
+      	 <h4>Semantic search: SPARQL</h4>
+      	 <span class="rfc2119-assertion" id="tdd-search-semantic-method"> The API MUST allow searching TDs using the SPARQL protocol [[sparql11-overview]]. The API MUST implement the protocol using <code>GET</code> requests, and MAY implement the protocol for <code>POST</code> requests.</span>
```

Comment:
The three assertions should probably be separated into three assertion blocks.

Reply:
I separated the assertion into three blocks as suggested in commit d718c1868e4a604a176ea72629ec4f686f0341ab